target_redshift/__init__.py

```diff
@@ -132,6 +140,14 @@ def add_metadata_values_to_record(record_message, stream_to_sync):
             "_sys_event_type", 1
     )
 
+    extended_record["_sys_transaction_lineno"] = record_message.get("record", {}).get(
+            "_sys_transaction_lineno", -1
```

Comment:
Default to -1 if not there. Should not happen but should set to something?

Reply:
Also would be helpful to send an alert if this isn't populated. And add a note somewhere that we are assuming that this will always be populated.

Reply:
i could setup a test is slack to alert us if a -1 passes through...

Reply:
It seems like a good test to plop into DBT... at least that's my intuition

Reply:
Which then could notify us via slack

Reply:
Circling back on this, I ended up creating a python module to send a slack alert in bi-data-integration. I'll share it soon

Reply:
blam thanks!